src/components/Main.tsx

```diff
@@ -2,34 +2,37 @@ import { Link, Outlet } from "react-router-dom";
 import { useAppServices } from "./AppServicesContext";
 import logo from "./logo.svg";
 import "./Main.css";
+import { SingletonEditorProvider } from "./SingletonEditor";
 
 export function Main() {
   const {
     appConfiguration: { loginPageUrl },
   } = useAppServices();
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <div className="flex-row-center h-full">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1>Editors WebApp</h1>
+    <div className="App dp-library dp-wrapper">
+      <header>header</header>
+      <main>
+        <SingletonEditorProvider>
+          <Outlet />
+        </SingletonEditorProvider>
+      </main>
+      <footer>
+        <div className="ed-cta-footer">
+          <button
+            type="button"
+            className="dp-button button-medium secondary-green"
+          >
+            Salir y editar luego
+          </button>
+          <button
+            type="button"
+            className="dp-button button-medium primary-green"
+          >
+            Continuar
+          </button>
         </div>
-        <nav className="flex-row-center">
-          <div>
-            <Link to="/invoices">Invoices</Link> |{" "}
-            <Link to="/expenses">Expenses</Link> |{" "}
-            <Link to="/campaigns/html123">campaigns/html123</Link> |{" "}
-            <Link to="/campaigns/html456">campaigns/html456</Link> |{" "}
-            <Link to="/campaigns/123">campaigns/123</Link> |{" "}
-            <Link to="/campaigns/456">campaigns/456</Link> |{" "}
-            <Link to="/campaigns/789">campaigns/789</Link> |{" "}
-            <Link to="/templates/1">/templates/1</Link> |{" "}
-            <a href={loginPageUrl}>Login</a>
```

Comment:
These links are being useful to test the application flow and detect unnecessary renders and requests to the backend.

Since the header is empty by the moment, what do you think about keeping these links there by the moment? 

<img width="819" alt="image" src="https://user-images.githubusercontent.com/1157864/155712598-9d01db7f-70d8-430b-8d64-92b111e7f155.png">

Reply:
👍